romana/openstack/tenant.go

```diff
@@ -208,8 +216,8 @@ func TenantExists(name string) bool {
 }
 
 // GetTenantUUID returns openstack tenant UUID corresponding to the name.
```

Comment:
The variable is called 'tenant', but you refer to the 'name' in the docstring. Maybe call the variable tenantName or change the docstring to "... the tenant, which is specified by its name."